### PR TITLE
Update to use conda build 3

### PR DESCRIPTION
**Pull request by marcelotrevisani**

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
@conda-forge-admin , please rerender

**Comment by marcelotrevisani** [comment]
This PR is just to try to move this recipe to conda build 3

**Comment by marcelotrevisani** [comment]
cc: @ocefpaf @jakirkham

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,34 +20,27 @@ source:
...
 requirements:
   build:
-    - toolchain
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake >=3.1
+  host:
     - python         # [win]
     - libtool        # [unix]
```
Comment (minrk) [inline recipe/meta.yaml]: libtool should also move to a build dependency

**Comment by minrk** [comment]
I made some comments that should get this closer, but I think this cannot proceed until toolchain_fort's run_exports problem is fixed: https://github.com/conda-forge/toolchain-feedstock/pull/39

**Comment by minrk** [comment]
@marcelotrevisani are you still working on this, or would you like me or someone to have a go at finishing it? The libgfortran fix has landed in toolchain, so should be adopted as soon as the next build is triggered.

**Comment by marcelotrevisani** [comment]
Hi @minrk , I'm kinda a busy these days because of my work, I think I will be able to change that just in the end of the week. However, feel free to do any modification which you want to. :)

**Comment by minrk** [comment]
@marcelotrevisani I added one more commit to this in #91 and I think it'll work.

**Pull request merged**